packages/css/src/components/header/README.md

```diff
@@ -12,12 +12,14 @@ Includes the name of the application if it is not the general website.
 - The Header is important because it conveys our corporate identity and is the first thing people see.
   Using it consistently helps users recognize and trust the website.
 - It is the same on every page of the application.
```

Comment:
Also for full page map type of things? The current and new Meldingen design doesn't follow this guideline atm

Reply:
We’ve updated the text.